README.md

```diff
@@ -48,6 +48,14 @@ using [lazy.nvim](https://github.com/folke/lazy.nvim):
       end,
       desc = 'gitpad branch',
     },
+    {
+      "<leader>pbv",
```

Comment:
Not that the example matters but maybe choose a different keybind for this since `<leader>pb` will wait for a couple milliseconds before opening as there is also `<leader>pbv`